package.json

```diff
@@ -49,6 +49,21 @@
       "require": "./lib/node/interceptors/RemoteHttpInterceptor.js",
       "import": "./lib/node/interceptors/RemoteHttpInterceptor.mjs",
       "default": "./lib/node/interceptors/RemoteHttpInterceptor.js"
+    },
+    "./presets/node": {
+      "browser": null,
+      "types": "./lib/node/preset/index.d.ts",
+      "require": "./lib/node/preset/index.js",
+      "import": "./lib/node/preset/index.mjs",
+      "default": "./lib/node/preset/index.js"
+    },
+    "./presets/browser": {
+      "browser": {
+        "types": "./lib/browser/preset/index.d.ts",
+        "require": "./lib/browser/preset/index.js",
+        "import": "./lib/browser/preset/index.mjs",
+        "default": "./lib/browser/preset/index.js"
+      }
```

Comment:
Was trying this out but having issues. I think it may be because this refers to `preset` singular in the path even though the actual path has `presets` plural?

Reply:
Ouch, I must've forgotten to fix this. Thanks for pointing out! Would you like to open a PR for this please?

Reply:
Sure thing. https://github.com/mswjs/interceptors/pull/361